test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -0,0 +1,109 @@
+﻿$codeCoverageZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/CodeCoverage.zip'
+$testContentZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/tests.zip'
+$openCoverZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/OpenCover.zip'
+
+
+New-Item "$PSScriptRoot\logs.txt"  -Force
```

Comment:
You might consider putting the log in some more typical output directory, or at least adding the file you create to `.gitignore`.